src/ctia/domain/entities.clj

```diff
@@ -34,44 +35,52 @@
                     (:end_time prev-valid-time)
                     time/default-expire-date)}})
 
+(defn default-realize
+  [type-name Model new-object id ident-map prev-object services]
```

Comment:
too many positional args

Reply:
I wrap these parameters in a map, but the point was to simply wrap the body of the delayed fn to simplify its testing.